flux_led/__main__.py

```diff
@@ -691,38 +702,78 @@ def close(self):
         finally:
             self._socket = None
 
-    def _determineMode(self, ww_level, pattern_code, mode_code):
-        mode = "unknown"
+    def _determineMode(self):
+        pattern_code = self.raw_state.preset_pattern
         if self.device_type == DeviceType.Switch:
             return "switch"
         if pattern_code == 0x61:
-            if mode_code == 0x01:
```

Comment:
This mostly changes `self.mode` back to what is was in 0.22 since I moved most of it into `color_modes`/`color_mode`.  Open to other ideas.